misc-scripts/xref_mapping/XrefParser/VGNCParser.pm

```diff
@@ -15,6 +15,63 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 
+=head1 CONTACT
+
+  Please email comments or questions to the public Ensembl
+  developers list at <http://lists.ensembl.org/mailman/listinfo/dev>.
+
+  Questions may also be sent to the Ensembl help desk at
+  <http://www.ensembl.org/Help/Contact>.
+
+=head1 NAME
+
+XrefParser::VGNCParser
+
+=head1 DESCRIPTION
+
+A parser class to parse the VGNC source.
+VGNC is the official naming source for some vertebrates species
+(currently chimp, cow, dog and horse).
```

Comment:
I would remove this bit and save ourselves the trouble of having to update it every time we add the parsing of VGNC data for another species - of which we will, according to an e-mail Bethan Yates sent to ensembl-core on 2019-06-10, get quite a few of soon. In fact, the above list is already out of date because we have had cat, macaque and mouse-lemur VGNC xrefs in Ensembl since release 98.